### ISSUE TITLE
Random "failures": "Agent process on a persistent worker exited unexpectedly!"

### ISSUE THREAD

**Issue by real-or-random** [cirruslabs/osx-images#33]
I'm seeing "Agent process on a persistent worker exited unexpectedly!"

https://cirrus-ci.com/task/4695778608283648
https://cirrus-ci.com/task/5716125398859776

Interestingly, the logs look file. It seems that everything is running smoothly but the task is marked as failed.

**Comment by fkorotkov** [comment]
Sorry about that! Unpredicted side effect of an update. Should be working now. We [just moved to another virtualization and overall architecture](https://medium.com/cirruslabs/new-macos-task-execution-architecture-for-cirrus-ci-604250627c94) and hitting some bumps. Re-ran your tasks

**Comment by real-or-random** [comment]
Hey, @fkorotkov, we're seeing this again for a few tasks, e.g.:
 https://cirrus-ci.com/task/4637194787749888

We also saw it yesterday on the same PR.

**Comment by fkorotkov** [comment]
Seems all the issue were with `d9-6` worker. I've checked it and it got the VM corrupted.

<img width="1127" alt="Screen Shot 2021-03-15 at 6 27 57 PM" src="https://user-images.githubusercontent.com/989066/111229449-4ef5a780-85bc-11eb-863f-4d6aed2e4725.png">

Unfortunately, it was happening for a not very popular image and therefor alert didn't fire. I've lowered the alert threshold so hopefully next time we'll fix it pro-actively. Sorry for the inconveniences!
